packages/xstate-wallet/src/ui/stories/application-workflow.stories.tsx

```diff
@@ -5,43 +5,38 @@ import {
 export default {title: 'X-state wallet'};
 import {storiesOf} from '@storybook/react';
 import {interpret} from 'xstate';
-import {renderWalletInFrontOfApp} from './helpers';
+import {renderComponentInFrontOfApp} from './helpers';
 import {MemoryStore} from '../../store/memory-store';
 import {MessagingServiceInterface, MessagingService} from '../../messaging';
+import React from 'react';
+import {ApplicationWorkflow} from '../application-workflow';
 
-const store = new MemoryStore(['0xkey']);
+const store = new MemoryStore([
+  '0x8624ebe7364bb776f891ca339f0aaa820cc64cc9fca6a28eec71e6d8fc950f29'
+]);
 const messagingService: MessagingServiceInterface = new MessagingService(store);
 const testContext = {
   channelId: '0x697ecf681033a2514ed19c90299a67ae8677f3c78b5877fe4550c4f0960e87b7'
 };
 
 if (applicationWorkflowConfig.states) {
   Object.keys(applicationWorkflowConfig.states).forEach(state => {
-    const machine = interpret<any, any, any>(
-      applicationWorkflow(store, messagingService).withContext(testContext),
-      {
-        devTools: true
-      }
-    ); // start a new interpreted machine for each story
-    machine.start(state);
-    storiesOf('Workflows / Application', module).add(
-      state.toString(),
-      renderWalletInFrontOfApp(machine)
-    );
-    machine.stop();
-  });
-}
-
-if (applicationWorkflowConfig.states) {
-  ['CREATE_CHANNEL', 'OPEN_CHANNEL'].forEach(event => {
-    const machineWithChildren = interpret<any, any, any>(
-      applicationWorkflow(store, messagingService).withContext(testContext)
-    ).start(); // start a new interpreted machine for each story
-    machineWithChildren.send(event);
-    storiesOf('Workflows / Application', module).add(
-      'Initialising + ' + event,
-      renderWalletInFrontOfApp(machineWithChildren)
-    );
-    machineWithChildren.stop();
+    // TODO: We should figure out a nice way of dealing with nested workflows
```

Comment:
Perhaps create a Github Issue for this instead of leaving as a comment (cc @junjizhi for recommending we avoid `TODO` in the code).